src/test/resources/features/mts/CaseDetails.feature

```diff
@@ -0,0 +1,30 @@
+@CaseDetails @MTS
+Feature: Case Details
+
+  Background:
+    Given I log in to DECS
+    And I create a single "MTS" case and return to the dashboard
+    And I load and claim the current case
+
+  Scenario: The user is able to complete and close an MTS Case
+    And I complete the Case Details stage and close the MTS Case
+    Then the case should be closed
+
+  Scenario: The user is able to put an MTS Case On Hold
+    And I put the MTS case on hold
+    Then the case should be moved to the "On Hold" stage
+
+  @Validation
+  Scenario Outline: User triggers error messages at the MTS Case Details stage
```

Comment:
I wonder if there is a more efficient way of doing this. It seems like we make a real chore of doing all the error messages individually, when we can trigger most of them in one go. The test runs are already pretty long. Should we consider triggering multiple and having an assertion that checks for the presence of multiple validation error messages in one go?